stubs/backports.ssl_match_hostname/backports/ssl_match_hostname/__init__.pyi

```diff
@@ -1,3 +1,5 @@
+from ssl import _PeerCertRetDictType
```

Comment:
I had two options:
1. Replicating the very same complex type
2. Using protected type alias from stdlib

I went with the `2`.